ci/jenkins/Test.groovy.j2

```diff
@@ -14,11 +21,19 @@
   test_method_names=test_method_names,
 ) %}
   {% if shard_index == 1 %}
-  {{ m.download_artifacts(tag='gpu2', filenames=tvm_multilib) }}
-  cpp_unittest(ci_gpu)
+  {{ m.download_artifacts(tag='gpu2', filenames=tvm_multilib + crttest + standalone_crt) }}
+  sh "${docker_run} --no-gpu ${ci_gpu} ./tests/scripts/task_config_build_gpu_other.sh build"
+  // These require a GPU to finish the build (i.e. CUDA needs to be load-able)
+  make_standalone_crt(ci_gpu, 'build')
+  // make_cpp_tests(ci_gpu, 'build')
+  // cpp_unittest(ci_gpu)
```

Comment:
These aren't run at all right now and are currently broken, so this leaves them commented out. The previous code uploaded the same binaries for both `gpu` and `gpu2` since there was a fixed reference to the `build` directory. This resolves the issue which causes this to fail but doesn't fix the build failure itself, which we should leave for a follow up